docs/api/topics/firefox_os_addon_submission.rst

```diff
@@ -94,6 +91,28 @@ Add-on Creation
 
     **Response**
 
-    An :ref:`add-on <addon-response-label>`.
+    The newly created :ref:`add-on <addon-response-label>`. Note that an add-on
+    version is created automatically for you.
 
     :status: 201 successfully created.
+
+Add-on Version Creation
+=======================
+
+.. _addon-version-post-label:
+
+.. http:post:: /api/v2/extensions/extension/(int:id)|(string:slug)/versions/
+
+    .. note:: Requires authentication, ownership of the add-on and a successful validation result.
+
+    **Request**
+
+    :param upload: the id of the :ref:`validation result <addon_validation>`
```

Comment:
Updated to use `validation_id` instead of `upload` (in both add-on and version creation endpoints)